### DISCUSSION TITLE
lax.dynamic_slice with (constant) stride

### DISCUSSION THREAD

**Discussion by PhilipVinc** [jax-ml/jax#5543] [Q&A]
Is there a way to implement some sort of `dynamic_slice` with a stride? apparently it's possible to do that only with non -dynamic start indices...

**Comment by hawkinsp** [comment]
My suggestion is to compute `slice(dynamic-slice(...), ..., stride)`. i.e., take a second strided slice on the output of the dynamic-slice. It's quite possible the compiler will fuse the two.
